packages/api/public/spec/index.yml

```diff
@@ -0,0 +1,21 @@
+openapi: 3.0.3
+info:
+  title: Gloss Translation API
+  description: |-
+    This is the Gloss Translation API. 
+    The frontend can be accessed at [https://gloss-translation.vercel.app]().
+  version: 0.0.1
+servers:
+  - url: https://gloss-translation-api.vercel.app/api
+    description: Production server
+  - url: http://localhost:4300/api
+    description: Local server
+paths:
+  /languages:
+    $ref: ./paths/languages.yml
+  /languages/{id}:
+    $ref: ./paths/language.yml
+components:
+  schemas:
+    Language:
+      $ref: ./schemas/language.yml
```

Comment:
This is the root open api spec, and we can import everything else from other files. Unfortunately, there is no good ways for having intermediate references to do things like collect all of the language paths in a single reference, so instead every component and path has to be referenced separately.